doschema/__init__.py

```diff
@@ -22,7 +22,48 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""JSON Schema utility functions and commands."""
+"""JSON Schema utility functions and commands.
+
```

Comment:
As doschema will not only do validation, the JSONSchemaValidator documentation should be in `Validation` -> `Compatibility validation`.